queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/RequestCommands/OutOfOperationReceiptCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Threading.Tasks;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Contracts.Constants;
+using fiskaltrust.Middleware.Contracts.Repositories;
+using fiskaltrust.Middleware.Localization.QueueDEFAULT.Factories;
+using fiskaltrust.storage.serialization.DE.V0;
+using fiskaltrust.storage.V0;
+using Newtonsoft.Json;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.RequestCommands
+{
+    public class OutOfOperationReceiptCommand : Contracts.RequestCommands.OutOfOperationReceiptCommand
```

Comment:
IMO this class hierarchy is very confusing. It makes it hard to understand the process and also hard to review. Currently I am not sure if we forget to set StopMoment or if it is set in the BaseCommand 🤔 Currently have no better idea for solving that but leaving this comment so that if anyone has an idea we can discuss that :)

Reply:
The idea was that everything which is common to all markets (maybe you can think of it as everything that would be documented in the general part of the interface-docs) is done in the base classes.

It does get hard to review. The idea is that if we give the abstract methods in the base class meaningful names it should be clear what still has to be done by the country specific command.

Reply:
Makes sense :)